<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Malakhov A.S. aka Brezerk</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>585</width>
    <height>497</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>%1 :. Qt GUI for Wine v%2</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout">
    <property name="leftMargin">
     <number>3</number>
    </property>
    <property name="topMargin">
     <number>3</number>
    </property>
    <property name="rightMargin">
     <number>3</number>
    </property>
    <property name="bottomMargin">
     <number>3</number>
    </property>
    <item row="0" column="0">
     <widget class="QTabWidget" name="tbwGeneral">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabPrograms">
       <attribute name="title">
        <string>Programs</string>
       </attribute>
       <widget class="QFrame" name="gbInfo">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>170</y>
          <width>311</width>
          <height>61</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="leftMargin">
          <number>2</number>
         </property>
         <property name="topMargin">
          <number>2</number>
         </property>
         <property name="rightMargin">
          <number>2</number>
         </property>
         <property name="bottomMargin">
          <number>2</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="lblIconInfo0">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string notr="true"/>
           </property>
           <property name="textFormat">
            <enum>Qt::RichText</enum>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="lblIconInfo1">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string notr="true"/>
           </property>
           <property name="textFormat">
            <enum>Qt::RichText</enum>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QLabel" name="lblIconInfo2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string notr="true"/>
           </property>
           <property name="textFormat">
            <enum>Qt::RichText</enum>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tabProcess">
       <attribute name="title">
        <string>Processes</string>
       </attribute>
       <layout class="QVBoxLayout" name="tabProcessLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
       </layout>
      </widget>
      <widget class="QWidget" name="tabPrefixSeup">
       <attribute name="title">
        <string>Setup</string>
       </attribute>
       <layout class="QGridLayout" name="setupLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
       </layout>
      </widget>
      <widget class="QWidget" name="tabPrefix">
       <attribute name="title">
        <string>Prefixes</string>
       </attribute>
       <layout class="QVBoxLayout" name="tabPrefixLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
       </layout>
      </widget>
      <widget class="QWidget" name="tabAppDB">
       <attribute name="title">
        <string>Wine AppDB</string>
       </attribute>
       <layout class="QVBoxLayout" name="tabAppDBLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
       </layout>
      </widget>
      <widget class="QWidget" name="tabLogging">
       <attribute name="title">
        <string>Logging</string>
       </attribute>
       <layout class="QGridLayout" name="logLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>585</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="mainRun"/>
    <addaction name="separator"/>
    <addaction name="mainOptions"/>
    <addaction name="separator"/>
    <addaction name="mainExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>He&amp;lp</string>
    </property>
    <widget class="QMenu" name="menuHandbook">
     <property name="title">
      <string>&amp;Handbook</string>
     </property>
     <addaction name="mainFirstSteps"/>
     <addaction name="mainFAQ"/>
     <addaction name="separator"/>
     <addaction name="mainIndex"/>
    </widget>
    <addaction name="mainAbout"/>
    <addaction name="mainAboutQt"/>
    <addaction name="separator"/>
    <addaction name="menuHandbook"/>
    <addaction name="mainWebsite"/>
    <addaction name="mainBugs"/>
    <addaction name="separator"/>
    <addaction name="mainDonate"/>
    <addaction name="separator"/>
    <addaction name="menuHelpThisTab"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="mainPrograms"/>
    <addaction name="mainProcess"/>
    <addaction name="mainSetup"/>
    <addaction name="mainPrefix"/>
    <addaction name="mainAppDB"/>
    <addaction name="mainLogging"/>
    <addaction name="separator"/>
    <addaction name="mainExportIcons"/>
    <addaction name="mainImportWineIcons"/>
    <addaction name="separator"/>
    <addaction name="mainImageManage"/>
    <addaction name="mainVersionManager"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="mainRun">
   <property name="text">
    <string>&amp;Run...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="mainExit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="mainAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="mainAboutQt">
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
  </action>
  <action name="mainInstall">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Installation Wizard</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="mainPrograms">
   <property name="text">
    <string>&amp;Programs</string>
   </property>
  </action>
  <action name="mainProcess">
   <property name="text">
    <string>P&amp;rocess</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="mainSetup">
   <property name="text">
    <string>&amp;Setup</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="mainPrefix">
   <property name="text">
    <string>Pre&amp;fixes</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="mainExportIcons">
   <property name="text">
    <string>&amp;Extract Images from PE Files</string>
   </property>
   <property name="statusTip">
    <string>Extract images from PE files using icoutils.</string>
   </property>
  </action>
  <action name="mainOptions">
   <property name="text">
    <string>&amp;Options</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="mainImageManage">
   <property name="text">
    <string>&amp;Disc Image Manager</string>
   </property>
  </action>
  <action name="mainFirstSteps">
   <property name="text">
    <string>&amp;First Steps</string>
   </property>
  </action>
  <action name="mainFAQ">
   <property name="text">
    <string>&amp;Short FAQ</string>
   </property>
  </action>
  <action name="mainIndex">
   <property name="text">
    <string>&amp;Index</string>
   </property>
  </action>
  <action name="mainWebsite">
   <property name="text">
    <string>&amp;Website</string>
   </property>
  </action>
  <action name="mainDonate">
   <property name="text">
    <string>&amp;Donate</string>
   </property>
  </action>
  <action name="mainBugs">
   <property name="text">
    <string>&amp;Bugs</string>
   </property>
  </action>
  <action name="mainAppDB">
   <property name="text">
    <string>&amp;Wine AppDB Browser</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="mainLogging">
   <property name="text">
    <string>&amp;Logging</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="menuHelpThisTab">
   <property name="text">
    <string>Help &amp;for This Tab</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="mainImportWineIcons">
   <property name="text">
    <string>&amp;Import Wine Desktop Files</string>
   </property>
   <property name="iconText">
    <string>Import Wine Desktop Files</string>
   </property>
   <property name="toolTip">
    <string>Scan for Wine desktop files and import them into Q4Wine's icon view.</string>
   </property>
   <property name="statusTip">
    <string>Scan for Wine desktop files and import them into Q4Wine's icon view.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="mainVersionManager">
   <property name="text">
    <string>Wine &amp;Version Manager</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="2" margin="3"/>
 <resources/>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>5</number>
  </property>
  <property name="gridDeltaY">
   <number>5</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
